TidyChat/Localization/Resources/Languages.resx

```diff
@@ -986,4 +986,10 @@ Initials: E. N.</value>
   <data name="ShowPartyObjectiveAndPartyCommentWhenJoiningAParty" xml:space="preserve">
     <value>Show Party Objective and Party Comment when joining a party</value>
   </data>
+  <data name="SystemTab_ShowEverythingElse" xml:space="preserve">
+    <value>Show everything else</value>
```

Comment:
For the record, I'm not super happy with this phrasing. Maybe "Show all other system messages" would be clearer?

Reply:
I agree. "Show all other System messages." The help text already clears up any confusion that this wording may cause, although I think even without the extra help text this wording is fairly straightforward. 

If you don't mind updating that I'll go ahead and merge the PR after as it LGTM. There's only a few more fixes left to make and then I'll be pushing 2.0 to Stable and out of Testing.